client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -460,32 +454,35 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         SELENIUM_RUN_LOCALLY("runLocally", "false", true),
 
         /**
-         * This parameter represents the folder which would contain the IOS app. This parameter is currently only useful
+         * This parameter represents the folder which would contain the mobile app. This parameter is currently only useful
          * for local runs. This is the folder from which applications would be searched for by SeLion when it comes to
-         * iOS automation in local runs. So please ensure that all the built applications reside in this folder. This is
-         * a mandatory value without which iOS native automation would not work. If no value is provided, SeLion assumes
-         * that the application is available under a directory named "Applications" in the current working directory.
+         * mobile automation in local runs. Please ensure that all the built applications reside in this folder.
+         * If no value is provided, SeLion assumes that the application is available under a directory named
+         * "Applications" in the current working directory.
          */
-        SELENIUM_NATIVE_APP_FOLDER("appFolder", System.getProperty("user.dir") + File.separator + "Applications", true),
+        MOBILE_APP_FOLDER("mobileAppFolder", System.getProperty("user.dir") + File.separator + "Applications", true),
```

Comment:
@sebady on 2nd read of your comment, I don't feel `MOBILE_APP_FOLDER` global-only should be `false`. When I read your comment the first time, for some reason I thought you were asking about `MOBILE_APP_NAME` which is already set to `false` for global-only